apps/web/src/pages/JobPosterTemplateAdminPages/UpdateJobPosterTemplatePage/components/JobDetailsSection/Display.tsx

```diff
@@ -0,0 +1,125 @@
+import { useIntl } from "react-intl";
+
+import { commonMessages } from "@gc-digital-talent/i18n";
+import { FragmentType, getFragment } from "@gc-digital-talent/graphql";
+import { Chip, Chips, Link } from "@gc-digital-talent/ui";
+
+import FieldDisplay from "~/components/FieldDisplay/FieldDisplay";
+import messages from "~/messages/jobPosterTemplateMessages";
+
+import { InitialData_Fragment } from "./JobDetailsSection";
+
+interface DisplayProps {
+  initialDataQuery: FragmentType<typeof InitialData_Fragment>;
+}
+
+const Display = ({ initialDataQuery }: DisplayProps) => {
+  const intl = useIntl();
+  const notProvided = intl.formatMessage(commonMessages.notProvided);
+
+  const {
+    name,
+    description,
+    supervisoryStatus,
+    workStream,
+    workDescription,
+    keywords,
+    classification,
+    referenceId,
+  } = getFragment(InitialData_Fragment, initialDataQuery);
+
+  // I don't want the name appended so not using the helper
+  const classificationString =
+    classification?.group && classification?.level
+      ? `${classification.group}-${classification.level.toString().padStart(2, "0")}`
+      : null;
+
+  const titleEn = `${classificationString} ${name?.en}`.trim();
+  const subtitleEn = `${supervisoryStatus?.label.en} in the ${workStream?.community?.name?.en}, ${workStream?.name?.en}`;
+
+  const titleFr = `${classificationString} ${name?.fr}`.trim();
+  const subtitleFr = `${supervisoryStatus?.label.fr} au ${workStream?.community?.name?.fr}, ${workStream?.name?.fr}`;
+
+  return (
+    <div className="grid gap-6 sm:grid-cols-2">
+      <h3 className="order-1 font-bold sm:order-1">
+        {intl.formatMessage({
+          defaultMessage: "English job details",
+          id: "elnOtk",
+          description: "Title for the English job details section",
+        })}
+      </h3>
+      <div className="order-2 sm:order-3">
+        <p className="font-bold text-primary-600 dark:text-primary-200">
+          {titleEn ?? notProvided}
+        </p>
+        <p className="text-sm text-gray-500 dark:text-gray-300">{subtitleEn}</p>
```

Comment:
This has a colour contrast error. I think for dark we need to go either `200` or `100`

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/issues/14268